test/Dialect/FIRRTL/SFCTests/GrandCentralInterfaces/Wire.fir

```diff
@@ -3,6 +3,7 @@
 ; RUN: firtool --annotation-file %s.anno.json --annotation-file %s.Extract.anno.json %s --annotation-file %s.YAML.anno.json | FileCheck %s --check-prefixes YAML
 ; RUN: firtool --split-verilog --annotation-file %s.anno.json %s -o %t.folder > %t  && cat %t.folder/MyView_companion.sv | FileCheck %s --check-prefixes MYVIEW_COMPANION
 ; RUN: firtool --annotation-file %s.anno.json --annotation-file %s.Prefix.anno.json %s | FileCheck %s --check-prefixes PREFIX
+; RUN: firtool --annotation-file %s.anno.json %s --disable-opt
```

Comment:
I did verify that just running with `--disable-opt` on this test does cause a failure. This failed (before I squashed these commits together) as so:

```
: 'RUN: at line 6';   /Users/schuylere/repos/github.com/llvm/circt/build/bin/firtool --annotation-file /Users/schuylere/repos/github.com/llvm/circt/test/Dialect/FIRRTL/SFCTests/GrandCentralInterfaces/Wire.fir.anno.json /Users/schuylere/repos/github.com/llvm/circt/test/Dialect/FIRRTL/SFCTests/GrandCentralInterfaces/Wire.fir --disable-opt
--
Exit Code: 1

Command Output (stderr):
--
/Users/schuylere/repos/github.com/llvm/circt/test/Dialect/FIRRTL/SFCTests/GrandCentralInterfaces/Wire.fir:91:5: error: Grand Central View "MyView" has an invalid leaf value (this must be a node)
    wire w : { uint : UInt<1>, vec : UInt<1>[2], multivec : UInt<1>[3][2], vecOfBundle : { uint : UInt<4>, sint : SInt<2>}[2], otherOther : { other : { uint : UInt<4>, sint : SInt<2>}}}
    ^

--

********************
********************
Failed Tests (1):
  CIRCT :: Dialect/FIRRTL/SFCTests/GrandCentralInterfaces/Wire.fir


Testing Time: 27.34s
  Unsupported      :   3
  Passed           : 513
  Expectedly Failed:   3
  Failed           :   1
FAILED: test/CMakeFiles/check-circt /Users/schuylere/repos/github.com/llvm/circt/build/test/CMakeFiles/check-circt 
```